### ISSUE TITLE
Voronoi should take into account x0 and y0

### ISSUE THREAD

**Issue by f0rr0** [FormidableLabs/victory#1031]
# Bugs and Questions

### Checklist

* [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

* [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

* [x] I am using the latest version of Victory

* [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

Voronoi Container only considers `y` or `x`. This is misleading with continuous data which has `x0` and `y0`

### Reproduction

https://codesandbox.io/s/4z6ww6w7m0

The individual bars are triggered onlzy if you pass the mid point of each horizontal bar. They should get triggered as soon as the cursor is over the bar

**Comment by boygirl** [comment]
@f0rr0 this is tricky, since a voronoi diagram calculates optimized trigger areas for single points. It works great for lines and points, but it is not ideal for bars. Currently, the voronoi diagrams are calculated for the point corresponding to the top of each bar (y). I could change this so that the voronoi diagram is calculated for the midpoint between y0 and y. that might make more sense for bar and area charts. If you want the cursor position to trigger an event only when you're over a bar, you are better off actually attaching events to the bars themselves using the `events` props rather than using `VictoryVoronoiContainer`.

**Comment by f0rr0** [comment]
I agree it’s more suited to discrete data but in this case there is a discrete point x0,y0 and I think it makes sense to include it in calculating the voronoi function

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by boygirl** [comment]
I think the best I can do is use the midpoint between y and y0, but I will make that change.
